library/include/rocblas-functions.h

```diff
@@ -2417,6 +2417,122 @@ rocblas_zsyr(rocblas_handle handle,
+    x         array of pointers storing the different vector x_i on the GPU.
+    @param[in]
+    incx      rocblas_int
+              specifies the increment for the elements of x.
+    @param[inout]
+    A         array of pointers storing matrices A on the GPU.
+    @param[in]
+    lda       rocblas_int
+              specifies the leading dimension of A.
+    @param[in]
+    batch_count rocblas_int
+                number of instances in the batch
+
+    ********************************************************************/
+
+ROCBLAS_EXPORT rocblas_status rocblas_ssyr_batched(rocblas_handle     handle,
+                                                   rocblas_fill       uplo,
+                                                   rocblas_int        n,
+                                                   const float*       alpha,
+                                                   const float* const x[],
+                                                   rocblas_int        incx,
+                                                   float* const       A[],
+                                                   rocblas_int        lda,
+                                                   rocblas_int        batch_count);
+
+ROCBLAS_EXPORT rocblas_status rocblas_dsyr_batched(rocblas_handle      handle,
+                                                   rocblas_fill        uplo,
+                                                   rocblas_int         n,
+                                                   const double*       alpha,
+                                                   const double* const x[],
+                                                   rocblas_int         incx,
+                                                   double* const       A[],
+                                                   rocblas_int         lda,
+                                                   rocblas_int         batch_count);
+
+/*! \brief BLAS Level 2 API
+
+    \details
+    xSYR_strided_batched performs the matrix-vector operations
+
+        A[i] := A[i] + alpha*x[i]*x[i]**T
+
+    where alpha is a scalar, vectors, and A is an array of
+    n by n symmetric matrices. For i = 1 , … , batch_count
+
+    @param[in]
+    handle    rocblas_handle.
+              handle to the rocblas library context queue.
+    @param[in]
+    n         rocblas_int
+    @param[in]
+    alpha
+              specifies the scalar alpha.
+    @param[in]
+    x         a pointer to the first vector x_i on the GPU.
+    @param[in]
+    incx      rocblas_int
+              specifies the increment for the elements of x_i.
+    @param[in]
+    stridex      specifies the pointer increment between vectors (x_i) and (x_i+1).
+    @param[inout]
+    A         pointer to the first matrix A on the GPU.
+    @param[in]
+    lda       rocblas_int
+              specifies the leading dimension of A.
+    @param[in]
+    strideA      stride from the start of one matrix (A_i) and the next one (A_i+1)
```

Comment:
Looks like we should be specifying that rocblas_stride is used for stride parameters

Reply:
Yes, maybe a separate PR and update all strided functions as I see others that don't indicate the type for strides?   some like GER strided are still rocblas_int but should we update to rocblas_stride now.   Other parameters in doxygen don't indicate the type either and we should probably make it all consistent.    I'll do syr now.

Reply:
I can do that as part of my refactor